src/Foundatio/Messaging/InMemoryMessageBus.cs

```diff
@@ -54,8 +54,11 @@ public InMemoryMessageBus(Builder<InMemoryMessageBusOptionsBuilder, InMemoryMess
                 Data = body
             };
 
-            SendMessageToSubscribers(messageData);
-            return Task.CompletedTask;
+            try {
+                await SendMessageToSubscribers(messageData);
+            } catch {
+                // swallow exceptions from subscriber handlers for the in memory bus
```

Comment:
I think I'd rather have the logging up here because then we catch all exceptions thrown from `SendMessageToSubscribers` instead of only the ones we awaited for subscription handlers, especially since if the code changes some day and a handler is removed we could be missing out on knowing that a collection was mutated.